apps/frontend/src/modules/auth/slices/actions.ts

```diff
@@ -54,4 +54,14 @@ const signUp = createAsyncThunk<
 	return user;
 });
 
-export { getAuthenticatedUser, signIn, signUp };
+const logout = createAsyncThunk<null, undefined, AsyncThunkConfig>(
+	`${sliceName}/logout`,
+	async (_payload, { extra }) => {
+		const { storage } = extra;
+		await storage.drop(StorageKey.TOKEN);
+
+		return null;
```

Comment:
Yeah, I wanted to ask about that. I think I either have to return null here, or give that async arrow function return type of `Promise<undefined>`, because if I put void as first generic argument for `createAsyncThunk`, eslint throws the error: `void is only valid as a return type or generic type argument` which is strange, because here void is actually a generic type argument. I guess I should stick with second approach then?